include/openPMD/backend/Attributable.hpp

```diff
@@ -434,7 +434,8 @@ AttributableInterface::setAttribute( std::string const & key, T value )
         && !attri.m_attributes.key_comp()(key, it->first) )
     {
         // key already exists in map, just replace the value
-        it->second = Attribute(value);
+        Attribute::resource val(value);
+        it->second = Attribute(std::move(val));
```

Comment:
Triggers the same issue:
```suggestion
        it->second = Attribute(std::move(value));
```

Reply:
Could you test if this one works:
```cpp
Attribute::resource val(std::move(value));
it->second = Attribute(std::move(vale));
```
or even:
```cpp
it->second = Attribute(Attribute::resource(std::move(value)));
```
That way would at least be without copying

Reply:
Good idea. I have no access to Cori due to maintenance today and will test it tomorrow.

Reply:
Tested it now, the explicit construction of the resource in place works as a work-around, too :)
Added also a comment :)